front/public/locales/fr/operationalStudies/manageTrainSchedule.json

```
@@ -78,6 +78,16 @@
   "pathfindingMissingParamsSimple": "Informations manquantes pour la recherche",
   "pathfindingNoState": "Aucune recherche itinéraire.",
   "pathfindingStatus": "Statut recherche d’itinéraire",
+  "pathfindingErrors": {
+    "not_found_in_blocks": "Non trouvé dans les blocs",
+    "not_found_in_routes": "Non trouvé dans les itinéraires",
+    "not_found_in_tracks": "Non trouvé dans les voies",
+    "incompatible_electrification": "Électrification incompatible",
+    "incompatible_loading_gauge": "Gabarit incompatible",
+    "incompatible_signaling_system": "Système de signalisation incompatible",
+    "not_enough_path_items": "Pas assez d'éléments de chemin",
+    "pathfinding_failed": "Recherche d'itinéraire infructueuse"
```

Quelques suggestions si ce sont ces messages qui apparaissent dans le front. Le but est d'aider l'utilisateur à fix son infra grâce à l'éditeur pour trouver un chemin, donc voici ce que je propose pour les messages (anglais à adapter aussi si on applique ces changements) :

```suggestion
    "not_found_in_blocks": "Canton manquant",
    "not_found_in_routes": "Itinéraire manquant",
    "not_found_in_tracks": "Voie manquante",
    "incompatible_electrification": "Électrification incompatible",
    "incompatible_loading_gauge": "Gabarit incompatible",
    "incompatible_signaling_system": "Système de signalisation incompatible",
    "not_enough_path_items": "Eléments manquants pour la recherche : Origine, Destination, Matériel roulant",
    "pathfinding_failed": "Aucun chemin trouvé"
```

---

Suggestion appliquée ! (La traduction anglaise a également été mise à jour.)